packages/webapp/public/locales/fr/translation.json

```diff
@@ -958,7 +958,7 @@
     "TITLE": "Inviter un utilisateur",
     "WAGE": "Salaire",
     "WAGE_ERROR": "Le salaire doit être un nombre décimal valide et non négatif",
-    "WAGE_RANGE_ERROR": "MISSING"
+    "WAGE_RANGE_ERROR": "Le salaire doit être un nombre positif inférieur à 999 999 999 999"
```

Comment:
It looks like there are three extra 9s..?
English: "Salary must be a positive number less than 999,999,999"

Reply:
Whoops, my bad - should be fixed now.